docs/self-hosted/config-options.md

```diff
@@ -339,6 +339,17 @@ WebSockets work reliably across multiple containers of Directus.
 
 Redis is required when you run Directus load balanced across multiple containers/processes.
 
+### Data Retention
+
+| Variable              | Description                                                                                                      | Default Value |
+| --------------------- | ---------------------------------------------------------------------------------------------------------------- | ------------- |
+| `RETENTION_ENABLED`   | Whether or not to enable custom data retention settings. `false` will not delete data.       |
+| `RETENTION_SCHEDULE`  | The frequency at which to check for removable records.                                                           | `0 * * * *`   |
```

Comment:
might be good to say in which format this needs to be specified and maybe also what the default values means. at least I wouldn't know 😄

Reply:
Resolved via dda2aa202245f850b44034e2237cbee886655366